snakemake/resources.py

```diff
@@ -58,6 +60,19 @@ def callable(wildcards, input, attempt, threads, rulename):
                     # Triggers for string arguments like n1-standard-4
                     except NameError:
                         return val
+                    except Exception as e:
+                        if not (
+                            isinstance(e, FileNotFoundError) and e.filename in input
+                        ):
+                            # Missing input files are handled by the caller
+                            logger.error(
```

Comment:
Good idea. Instead of logging manually here, can you please raise a `WorkflowError` with that message? Snakemake will then handle the display properly.

Reply:
Thanks! The most recent commit will raise a `WorkflowError` instead of logging the error message.